config/environments/test.rb

```diff
@@ -26,8 +28,6 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Print deprecation notices to the stderr.
-  config.active_support.deprecation = :stderr
-
-  config.active_support.test_order = :random
```

Comment:
The default is now `:random`

> config.active_support.test_order sets the order that test cases are executed. Possible values are :random and :sorted. This option is set to :random in config/environments/test.rb in newly-generated applications. If you have an application that does not specify a test_order, it will default to :sorted, until **Rails 5.0, when the default will become :random.**

But I'll add it back